### PR TITLE
Set service config default values

### PR DESCRIPTION
**Pull request by aounhaider1**
Set default values of service parameters as defined in config description. The values are applied if no user does not specify a value. The related issue is https://github.com/eclipse/smarthome/issues/1051.
Signed-off-by: Aoun Bukhari <bukhari@itemis.de>

### CODE REVIEW

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/include.config.html [hunk 1]
```diff
@@ -7,18 +7,18 @@
...
+                                    <div ng-message="min">Minimum allowed number is {{parameter.min}}</div>
+                                    <div ng-message="max">Maximum allowed number is {{parameter.max}}</div>
+                                    <div ng-message="required">Field is required</div>
+                                </div>
+                                <span ng-messages="" ng-show="!focus">
+                                    <p class="hint">
+                                        <small>{{parameter.description}}</small>
+                                    </p>
+                                </span> </md-input-container>
+                            </div>
```
Comment (maggu2810) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/include.config.html]: There is a problem in the current IDE setup. See https://github.com/eclipse/smarthome/pull/1161#issuecomment-195974025 which one are you using? HTML files should use tabs and not spaces.

File: extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/include.config.html [hunk 2]
```diff
@@ -7,18 +7,18 @@
...
-									<div ng-message="max">Maximum allowed number is {{parameter.max}}</div>
-									<div ng-message="required">Field is required</div>
-								</div>
-								<span ng-messages="" ng-show="!focus">
-									<p class="hint">
-										<small>{{parameter.description}}</small>
-									</p>
-								</span> </md-input-container>
-							</div>
+                                <md-input-container> <label>{{parameter.label}}</label> <input ng-init="focus=false" name="{{parameter.name}}" type="number" min="{{parameter.min}}" max="{{parameter.max}}" ng-model="configuration[parameter.name]" ng-required="parameter.required" type="{{parameter.inputType}}" ng-focus="focus=true" ng-blur="focus=false">
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/include.config.html]: something is messed up with the formatting here - the html files should have tabs.

**Comment by aounhaider1** [comment]
I used https://github.com/eclipse/smarthome/pull/1015#issuecomment-187751734 settings, which seem to be wrong, will update it. And I believe we are using single tab, right?

**Comment by kaikreuzer** [comment]
tested successfully, thanks!

**Pull request merged**